lib/blocklist.c

```diff
@@ -6,22 +6,24 @@
  * of the License at http://www.apache.org/licenses/LICENSE-2.0
  */
 
-#include "blocklist.h"
 
 #include <errno.h>
 #include <stdio.h>
 #include <assert.h>
 #include <stdint.h>
 #include <stdlib.h>
+#include <stdbool.h>
 #include <string.h>
+#include "xalloc.h"
```

Comment:
I think that this move isn't correct since this is ZMap code, not a stdlib include.

Reply:
Ahh whoops, assumed that was where `malloc`/`calloc`/etc. came from. I'll move it back!